content/influxdb/cloud-dedicated/admin/tokens/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Manage tokens
+seotitle: Manage tokens in InfluxDB Cloud Dedicated
+description: >
+  Manage database tokens in your InfluxDB Cloud Dedicated cluster.
+  A database is a named location where time series data is stored.
+  Each InfluxDB database has a retention period, which defines the maximum age
+  of data stored in the database.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB
+weight: 101
+influxdb/cloud-dedicated/tags: [tokens]
+---
+
+InfluxDB uses token authentication to authorize access to data in your InfluxDB
+Cloud Dedicated cluster. Each token grants read and write permissions to one or
+more databases and allow for actions like writing and querying data.
+
+All read and write actions performed against time series data in your InfluxDB
+Cloud Dedicated cluster must be authorized using a token. Administrative actions
+such as managing tokens and databases are authorized using **Auth0** tokens, 
```

Comment:
We're referring to them as "Auth0 tokens", not "management tokens"?

Reply:
Oh, good catch. I'll mention management tokens heres.